ui/hooks/gasFeeInput/useGasFeeInputs.js

```diff
@@ -318,24 +333,10 @@ export function useGasFeeInputs(
     gasWarnings,
     hasGasErrors,
     hasSimulationError,
+    minimumGasLimit: hexToDecimal(minimumGasLimit),
```

Comment:
Seems like we don't have a ton of consistency in which base we pass these gas values around in so would be good to specify the base in the variable name?
```suggestion
    minimumGasLimitDec: hexToDecimal(minimumGasLimit),
```